README.md

```diff
@@ -31,63 +42,153 @@ envs:
 - A_SECRET_PARAM_TWO: the value for secret two
 ```
 
-## How to create your own step
+## Simple usage
 
-1. Create a new git repository for your step (**don't fork** the *step template*, create a *new* repository)
-2. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-3. Fill the `step.sh` with your functionality
-4. Wire out your inputs to `step.yml` (`inputs` section)
-5. Fill out the other parts of the `step.yml` too
-6. Provide test values for the inputs in the `bitrise.yml`
-7. Run your step with `bitrise run test` - if it works, you're ready
+### Example using public IDs
 
-__For Step development guidelines & best practices__ check this documentation: [https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md).
+<!-- TODO: change git urls to step references after we publish it -->
+```yml
+- git::https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git:
+   inputs:
+   - api_key: $DATADOG_API_KEY
+   - app_key: $DATADOG_APP_KEY
+   - public_ids: '012-345-678, 9ab-cde-fff'
```

Comment:
I'm not a fan of the "only-numbers" one... can you use the same example as the other integrations?

```suggestion
   - public_ids: 'abc-d3f-ghi, jkl-mn0-pqr'
```

Reply:
Will do, i released i changed them based on a false premise 🤦